_sass/components/_toc.scss

```diff
@@ -0,0 +1,43 @@
+$gray: #737373;
+$light-gray: lighten($gray, 52%);
+$dark-gray: darken($gray, 20%);
+
+#documentation *:first-child {
+  margin-top: 0;
+}
+
+.toc {
+  display: none !important;
+  padding: 0;
+
+  @media (min-width: 768px) {
+      display: block !important;
```

Comment:
this is super nitpicky, but I'm wondering if there's a reason the inner blocks have 4 space indentation while the outer blocks have 2?